methods/did:.md

```diff
@@ -0,0 +1,33 @@
+# did:*
+
+The "any" DID method encoding allows any DID to be represented as a multidid by essentially putting the entire DID ASCII string in the URL portion of the multidid byte string.
```

Comment:
I would personally expect that everything but `path`, `query` and `fragment` to go here instead, that way `Location` will be the same across all of the DID methods.

Reply:
But then we would need two length varints. One for the DID itself and one for the URL component. Seems less efficient?

Reply:
You are correct it would have some overhead when you actually have path / query / fragment. On the flip-side you'll have more uniform structure.

If you don't want to pay that overhead I would define `multidid` as follows instead

```ipldsch
type MultiDID union {
  | DID   "0x0d1d"
} representation varintprefix

type DID union {
  Key          "0xe7"
  # ....
  # did:pkh
  PKH         "0xca"
  # did:*
  Raw         "0x55"
} representation varintprefix
```

That way `Key`, `PKH` can have method bytes and url bytes, while `Raw` does no.